doc/user/content/transform-data/patterns/temporal-filters.md

```diff
@@ -9,25 +9,48 @@ menu:
     parent: 'sql-patterns'
 ---
 
-A _temporal filter_ is a `WHERE` or `HAVING` clause which uses the [`mz_now()`](/sql/functions/now_and_mz_now) function.
-This function returns Materialize's current virtual timestamp, which works to keep up with real time as data is processed.
-Applying a temporal filter reduces the working dataset, saving memory resources and focusing results on the recent past.
-
-Here is a typical temporal filter that considers records whose timestamps are within the last 5 minutes.
+A **temporal filter** is a query condition/predicate that filters data based on
+a time-related (i.e., temporal) condition. Using a temporal filter reduces the
```

Comment:
I think it's more specific than "time-related". It is specifically only *predicates* that involve `mz_now()`.

Reply:
Will update.